Makefile

```diff
@@ -27,6 +27,9 @@ flow:
 fix:
 	./node_modules/.bin/eslint packages/ --format=codeframe --fix
 
+prettify:
+	./node_modules/.bin/prettier --trailing-comma all --write "packages/**/src/*.js"
```

Comment:
@loganfsmyth the way prettier works is that it tries to fit as many things as possible within a single line. Usually, this is what you want for 80 columns. The main reason people extend it further is that sometimes you want to go beyond 80 columns, but prettier is going to do it all the time and makes things less readable in practice.